server/routes/notification.js

```diff
@@ -0,0 +1,13 @@
+const express = require("express");
+const router = express.Router();
+const protect = require("../middleware/auth");
+const { createNotification, getAllNotification, getUnreadNotification, updateNotification, updateAllUnreadNotification } = require("../controllers/notification");
+
+router.route("/create").post(protect, createNotification);
+router.route("/all").get(protect, getAllNotification);
+router.route("/").get(protect, getUnreadNotification);
+router.route("/:id").put(protect, updateNotification);
```

Comment:
Remember to use a `PATCH` request if you're not updating the whole resource.

Reply:
Thank for catching this, I will remember it in future PR.